Makefile

```diff
@@ -19,7 +19,9 @@ RPM_DEFINES := $(foreach v, _sourcedir _specdir _builddir _srcrpmdir _rpmdir ver
 
 DIST_DOM0 ?= fc13
 
+ifeq ($(DISTFILES_MIRROR),)
 DISTFILES_MIRROR := http://ftp.qubes-os.org/distfiles/
```

Comment:
Better use `?=` in place of `:=`, instead of `ifeq`.

Reply:
Indeed...